### ISSUE TITLE
New företagsbiljetter arent selectable when attending a showing

### ISSUE THREAD

**Issue by Edholm** [cthdidIT/itbio#196]
If I add a new ftg ticket and then want to use that same ticket when attending a showing, it doesn't show up in the drop-down list.

**Comment by lindskogen** [comment]
Should it?

Ideally, the ftg tickets shouldn't expire before `expectedBuyDate`, maybe we change it to that instead, if we implement that?

**Comment by Edholm** [comment]
Yes, that is how I implemented it in the backend. But if the ticket expires today and I will book the showing today, that should be fine right?
